### PR TITLE
Tidy up main_loop, removing namespace pollution

### PR DESCRIPTION
**Pull request by bluewavenet**
Signed-off-by: Rob White <rob@blue-wave.net>

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -355,10 +359,10 @@ main_loop(void)
 			exit(1);
 		}
 	}
-
 	/* TODO: set listening socket */
 	debug(LOG_NOTICE, "Created web server on %s", config->gw_address);
 
+	/* If login script is enabled, check if the script actually exists */
```
Comment (mwarning) [inline src/main.c]: We should do one line comments in one style. `/**/` or `//`

File: src/main.c [hunk 2]
```diff
@@ -229,15 +229,13 @@ init_signals(void)
 }
 
 /**@internal
- * Main execution loop
+ * Setup from Configuration values
  */
 static void
-main_loop(void)
+setup_from_config(void)
```
Comment (mwarning) [inline src/main.c]: Maybe we should pass the config variable here as an argument.
I do not have a strong opinion here.

File: src/main.c (hunk 1 above)
Comment (bluewavenet) [reply src/main.c]: Yes, I agree. As I am in favour of having lots of comments, I think having a recommended style is a good thing. My preferred use is // for single line comments.

Comment (mwarning) [reply src/main.c]: then let's use `//` from here on :)

**Pull request merged**